benchmarks/distribution.py

```diff
@@ -60,25 +54,28 @@ class Functions:
     Benchmark various functions of distributions.
     """
 
-    param_names = ["dist", "property"]
-    params = [distribution_names, ["pdf", "logpdf", "cdf", "logcdf"]]
+    param_names = ["dist", "method"]
+    params = [DISTRIBUTION_NAMES, ["pdf", "logpdf", "cdf", "logcdf"]]
 
-    def setup(self, dist, property):
+    def setup(self, dist, method):
+        # pylint: disable=missing-function-docstring,attribute-defined-outside-init
+        # pylint: disable=unused-argument
```

Comment:
```python
# pylint: disable=unused-argument
# pylint: disable=missing-function-docstring
```
will appear in almost every benchmark and should if possible be ignored for all benchmarks.

Reply:
The docstring warning makes sense to be ignored globally IMO (the docstrings seem to be super repetitive anyway: "Time method xyz", "Benchmark memory usage of xyz". The `unused-argument` I am more hesitant about...

Reply:
I put the ignore for unused-argument back into the tox file. Any idea where we can best document that this is not one of the "we have not resolved this yet" ignores but a "final" ignore? I put a comment into the tox ini file, but if you have a better idea please share.

Reply:
Maybe @nathanaelbosch knows a way to ignore specific warnings in a certain folder, however judging from [StackOverflow](https://stackoverflow.com/questions/36182847/pylint-disable-specific-warnings-for-specific-folder) one can only achieve this by having a `.pylintrc` file in `benchmarks` and then having two `pylint` statements (which we already have anyway).

Reply:
@marvinpfoertner also looked into it and, if I'm not mistaken, came to the conclusion that it's unfortunately not possible to have configs on a per-directory basis.

In general I'm definitely open for ideas to move these exceptions out of the `tox.ini`, since the current way does not feel ideal.